samples/TagHelperSample.Web/Views/Home/Index.cshtml

```diff
@@ -24,6 +24,7 @@
 <script>bar();</script>
 
 <h2>Index</h2>
+<a asp-action="Index" asp-Controller="Admin" asp-area="Admin" asp-route-area="Yolo">Go to admin!</a>
```

Comment:
asp-area always win over asp-route-area. This behavior is similar to asp-action over asp-route-action